content/guide/setup.md

```diff
@@ -69,19 +83,19 @@ mkdir Quire
 
 *Less steps than for a Mac, but the Hugo one in particular, is a bit tricky.*
 
-1.) Download and install [Node](https://nodejs.org/en/download/). You will have to restart your computer for Node to finish installing.
+1.) Download and install **[Node.js 8.9.3.](https://nodejs.org/en/download/)**. You will have to restart your computer for Node to finish installing.
```

Comment:
Rather than specifying a specific release, I think we want them installing the current LTS version of Node, which stands for Long Term Support. When you worked on the docs it was probably 8.9.3, but now it's 8.9.4. Worth checking with @egardner on this.

Reply:
Ok